doc/releases/changelog-dev.md

```diff
@@ -175,9 +178,9 @@ Renke Huang,
 Soran Jahangiri,
 Christina Lee,
 Austin Huang,
-Christina Lee,
```

Comment:
Did you cancel Christina Lee (@albi3ro) from the list of contributors? I cannot checkout this branch (so I can't verify), but if that's the case please re-insert her name

Reply:
I see. All good then :+1: